src/rgw/rgw_multi.h

```diff
@@ -124,16 +124,16 @@ extern int list_multipart_parts(rgw::sal::RGWRadosStore *store, struct req_state
                                 bool assume_unsorted = false);
 
 extern int abort_multipart_upload(rgw::sal::RGWRadosStore *store, CephContext *cct, RGWObjectCtx *obj_ctx,
-                                RGWBucketInfo& bucket_info, RGWMPObj& mp_obj);
+                                RGWBucketInfo& bucket_info, RGWMPObj& mp_obj, const Span& parent_span = nullptr);
```

Comment:
would encourage use of nullptr for other instances as well

Reply:
noted